modules/ocf/manifests/packages/firefox.pp

```diff
@@ -0,0 +1,19 @@
+class ocf::packages::firefox {
+  package {
+    'firefox':
+      ensure => present;
+    'iceweasel':
+      ensure => absent;
+  }
+
+  file {
+    # disable caching, history, blacklisting, and set homepage
+    '/etc/firefox/profile/prefs.js':
```

Comment:
From [Debian Bug #815006](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=815006#110):

> Speaking of conffiles, version 46 is removing /etc/$package/profile.
> I'm very tempted to remove it in 45 because that would only leave
> /etc/iceweasel/pref/*.js to migrate, and would keep upgrade code free of
> conffiles removal entirely. This would allow to simply use
> /etc/firefox{,-esr}/*.js files.

Looks like it was changed with Firefox 45. No idea about where to put `localstore.rdf` or if we have an alternative way to always open maximized.